pyproject.toml

```
@@ -76,7 +76,7 @@ all-cuda12 = ['gt4py[cuda12,dace,formatting,jax-cuda12,performance,testing]']
 # Other extras
 cuda11 = ['cupy-cuda11x>=12.0']
 cuda12 = ['cupy-cuda12x>=12.0']
-dace = ['dace>=0.16.1', 'sympy>=1.9,<1.13']  # see https://github.com/spcl/dace/pull/1620
+dace = ['dace>=1.0.0,<=1.0.0']
```

```suggestion
dace = ['dace==1.0.0']
```
but not constraining the upper version should be better, why did you add the upper bound?

---

Updated in latest commit with explanation.